docs/cody/clients/install-cli.mdx

```diff
@@ -0,0 +1,212 @@
+# Install Cody CLI
+
+<p className="subtitle">Learn how to install the <code>cody</code> command-line tool.</p>
+
+<Callout type="note" title="Experimental">
+Cody CLI support is in the experimental stage.
```

Comment:
We should also mention what pricing tiers will it support in the callout.

Reply:
I added

> Cody CLI is available to Free, Pro, and Enterprise customers.

The `--context-repo` section highlights that this specific feature is limited to Enterprise users (just like in the IDE)